### ISSUE TITLE
 The problem involves PyTorch, but there's no error message; it just stops working. 

### ISSUE THREAD

**Issue by stevanisya** [comfyanonymous/ComfyUI#3665]
My friend and I encountered the same issue with ComfyUI on the same day.
Has anyone else experienced this problem? and anyone know how to resolve this problem ? Thanks :)

`transform` ToTensor()is deprecated and will be removed in a future release. Instead, please usev2.Compose([v2.ToImage(), v2.ToDtype(torch.float32, scale=True)]).Output is equivalent up to float precision. warnings.warn( Requested to load CLIPVisionModelProjection Loading 1 new model C:\Users\ComfyUI\comfy\ldm\modules\attention.py:357: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.) out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False) Requested to load SD1ClipModel Loading 1 new model Requested to load AutoencoderKL Loading 1 new model [AnimateDiffEvo] - INFO - Sliding context window activated - latents passed in (50) greater than context_length 16. Requested to load ControlNet Requested to load BaseModel Requested to load ControlNet Loading 3 new models 0%|                                                                                            | 0/8 [00:00<?, ?it/s]C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [32,0,0] Assertion -sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [33,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [34,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [35,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [36,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [37,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [38,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [39,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [40,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [41,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [42,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [43,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [44,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [45,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [46,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [47,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [48,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [49,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [50,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [51,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [52,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [53,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [54,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [55,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [56,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [57,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [58,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [59,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [60,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [61,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [62,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds"failed. C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\IndexKernel.cu:92: block: [502,0,0], thread: [63,0,0] Assertion-sizes[i] <= index && index < sizes[i] && "index out of bounds" failed. [ComfyUI launcher] Process got interupted, most likely due to an error.`

**Comment by ltdrdata** [comment]
It looks like your PyTorch installation is incorrect. What is your GPU and how did you install it?

**Comment by stevanisya** [comment]
my GPU is NVDIA GeForce RTX 4070 Ti, and I tried to uninstall and install PyTorch, still the same problem :(

**Comment by brayevalerien** [comment]
I'm the said friend. I got similar results on two different workstations setup similarly:
- OS: Windows 10 (@stevanisya is using Windows 11)
- Python: 3.10.11 (running in Conda environment)
- Pytorch version: 2.3.1 compiled for CUDA 11.8 (tested CUDA 12.1 and Pytorch nightly as well, without success)
- GPU: Quadro RTX 6000

After more testing, the issue appears when trying to run this workflow (using any model and any image/video): [workflow.json](https://github.com/user-attachments/files/15634919/workflow.json).
Default workflow works. Removing all extensions except the ones needed still yield the error messages.

**Comment by The-Administrator** [comment]
The version update is too fast（😔)

**Comment by mcmonkey4eva** [comment]
The error is likely caused by one of the extensions used in that workflow then? Potentially `AnimateDiffEvo` as it's mentioned just before the error starts in the log

(if you can replicate with just comfy default nodes we can reopen this issue and investigate)

**Comment by yurigushiken** [comment]
Hi there. I got the same errors . it happened randomly and suddenly on workflow that worked all week up to this point. I also get many errors stacked. Here is one of them:

ERROR lora diffusion_model.input_blocks.4.1.transformer_blocks.0.attn2.to_k.weight shape '[640, 768]' is invalid for input of size 1310720

**Comment by mcmonkey4eva** [comment]
@yurigushiken that is a different error, please post your own separate `User Support` issue

**Comment by yurigushiken** [comment]
Thank you for your response.
I was getting long strings of errors however, after refreshing my browser
many times to "clear the cache" I've got it to work again.

**Comment by yurigushiken** [comment]
Actually the problem was using the wrong number of frames for the frame
load cap.
Certain integers do not work and I did not know that.
On default settings, 8,16,32 frames work, but 20 frames crashes the system.
Probably many people know this I did not. Hope this helps if anyone has
this issue
